project-koku/koku
# COST-1133 - Sources delete task

This PR moves the source deletion into a celery task.

Testing:

---
1. create source
2. delete source. observe deletion taking place within a worker pod
---
1. create source
2. ingest large amt of data (large enough so that deletion takes some time)
3. delete the source.
4. kill the sources-client and restart
5. observe that the sources-client adds the source delete to the task queue again.
6. observe that the worker does not fail when the it tries to delete the same source again.
